subsys/mgmt/mcumgr/grp/img_mgmt/src/img_mgmt_state.c

```diff
@@ -325,66 +323,104 @@ img_mgmt_state_confirm(void)
 	return rc;
 }
 
+#define REPORT_SLOT_ACTIVE	BIT(0)
+#define REPORT_SLOT_PENDING	BIT(1)
+#define REPORT_SLOT_CONFIRMED	BIT(2)
+#define REPORT_SLOT_PERMANENT	BIT(3)
```

Comment:
Put defines near top of file before functions

Reply:
Moved up